cypress/e2e/tests/pages/manager/repositories.spec.ts

```diff
@@ -2,252 +2,271 @@ import PromptRemove from '@/cypress/e2e/po/prompts/promptRemove.po';
 import ChartRepositoriesPagePo from '@/cypress/e2e/po/pages/chart-repositories.po';
 import * as path from 'path';
 import * as jsyaml from 'js-yaml';
-import { LONG_TIMEOUT_OPT } from '@/cypress/support/utils/timeouts';
+import { EXTRA_LONG_TIMEOUT_OPT, LONG_TIMEOUT_OPT } from '@/cypress/support/utils/timeouts';
 
 const chartBranch = 'release-v2.9';
 
 describe('Cluster Management Helm Repositories', { testIsolation: 'off', tags: ['@manager', '@adminUser'] }, () => {
   const repositoriesPage = new ChartRepositoriesPagePo(undefined, 'manager');
   const downloadsFolder = Cypress.config('downloadsFolder');
+  const testRepos = [];
 
   before(() => {
     cy.login();
   });
 
-  beforeEach(() => {
-    cy.createE2EResourceName('repo').as('repoName');
-  });
+  describe('Create', () => {
+    beforeEach(() => {
+      const name = `e2e-repo-${ Math.random().toString(36).substr(2, 6) }`;
```

Comment:
Looking at some failed/repeated sorry cypress runs I noticed that the name created by `createE2EResourceName` was the same when tests were retried, leading tests that involved creating a resource with repoName to error out on all subsequent retries.

I wasn't sure if that behavior was an intentional and/or necessary aspect of `createE2EResourceName` so I didn't modify that method. If it is intentional perhaps we need another convenience method to randomly generate names that _wont_ be repeated on test retry.